scripts/setup/configure-certs

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+
+set -e
+if [ "$EUID" -ne 0 ]; then
+    echo "Error: This script must be run as root" >&2
+    exit 1
+fi
+
+# Server-specific construct
+if [ -z "$SERVER_NAME" ]; then
+    echo "No server name given."
+    SERVER_NAME="A boson of server"
```

Comment:
We should probably just `exit 1` here rather than generating a certificate with a weird organization ID, right?  And output some help output to tell the user what they need to do?